app/screens/gallery/index.tsx

```diff
@@ -28,26 +28,14 @@ type Props = {
     items: GalleryItemType[];
 }
 
-const styles = StyleSheet.create({
-    container: {
-        flex: 1,
-    },
-});
-
 const GalleryScreen = ({componentId, galleryIdentifier, hideActions, initialIndex, items}: Props) => {
     const dim = useWindowDimensions();
     const isTablet = useIsTablet();
     const [localIndex, setLocalIndex] = useState(initialIndex);
     const {headerAndFooterHidden, hideHeaderAndFooter, headerStyles, footerStyles} = useGalleryControls();
     const galleryRef = useRef<GalleryRef>(null);
 
-    const containerStyle = useMemo(() => {
-        if (Platform.OS === 'ios') {
-            return dim;
-        }
-
-        return styles.container;
-    }, [dim]);
+    const containerStyle = useMemo(() => dim, [dim]);
```

Comment:
This memo is doing nothing as far as I can see. We can do just the following:

```suggestion
    const containerStyle = dim;
```